pkg/kubectl/cmd/get_test.go

```diff
@@ -225,10 +225,10 @@ func TestGetUnknownSchemaObjectListGeneric(t *testing.T) {
 		}
 		arr := out["items"].([]interface{})
 		if arr[0].(map[string]interface{})["apiVersion"] != test.testtypeVersion {
```

Comment:
save locally and reuse, rather than repeating the index+cast:
```
if version := arr[0].(map[string]interface{})["apiVersion"]; version != test.testtypeVersion {
  ... version ...
}
```